consensus/logical/group_info.go

```diff
@@ -99,7 +100,7 @@ func (sgi StaticGroupInfo) GetMinerPos(id groupsig.ID) int {
 		pos = v
 		// Double verification
 		if !sgi.GInfo.Mems[pos].IsEqual(id) {
-			panic("double check fail!id=" + id.GetHexString())
+			pos = -1
```

Comment:
should panic. index different from mem-array position, somothing error

Reply:
done